docs/02-database.ipynb

```diff
@@ -0,0 +1,8574 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Add new record it to the collections ...

Add new record to the collections ...
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/eoapi-fedgeoday25-workshop/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>